src/components/ModalWithForm.js

```diff
@@ -5,8 +5,13 @@ export default class ModalWithForm extends Modal {
     super(modalSelector);
     this._handleFormSubmit = handleFormSubmit.bind(this);
     this._modalForm = this._modalElement.querySelector(".modal__form");
+    this._submitButton = this._modalElement.querySelector(
+      ".modal__save-button"
+    );
   }
 
+  variableTest() {}
+
   _getInputValues() {
     const inputs = this._modalForm.querySelectorAll(".modal__input");
```

Comment:
**Could be improved**

All elements found **every time** you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** in order not to waste resources on searching them again and again.

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form`, `this._likeButton`, and so on.